.github/workflows/CI-BUILD.yml

```
@@ -0,0 +1,172 @@
+jobs:
+  BUILD:
+    permissions:
+      actions: read
+      contents: read
+      statuses: write
+      packages: none
+      pull-requests: read
+      security-events: none
+    if: ${{ !cancelled() && (github.repository == 'reactive-firewall/multicast') }}
+    runs-on: ubuntu-latest
+    environment: ${{ env.ENVIRONMENT }}
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)